docs/docker/nvidia_docker/setup_nvidia_support.rst

```diff
@@ -0,0 +1,153 @@
+======================================
+Nvidia support for docker
+======================================
+.. _docker-nvidia-support-how-to:
+
+If you want to expose your nvidia driver into the docker container this is possible by using the `NVIDIA Container Toolkit <https://docs.nvidia.com/datacenter/cloud-native/container-toolkit/overview.html>`_. 
+
+Prerequisites
+""""""""""""""
+Before we can install the NVIDIA Container Toolkit we have to install the nvidia drivers for our platform and install docker.
+
+Nvidia drivers
+----------------
+Make sure you have the NVIDIA drivers for your Linux distribution installed. This can either be done by using your package manager or you can download the ``.run`` installers from NVIDIA Driver Downloads `here <https://www.nvidia.com/Download/index.aspx?lang=en-us>`_. This depends on the operating system and package manager you are using so you have to look it up.
```

Comment:
Please give concrete options here for Ubuntu.

Also we should add here a note about our internal computer setup. Should we add htis somewhere in the Nextcloud? The Nvidia drivers are automatically installed on our Tuxedo computers  when using Tuxedo Tomte.